frontend/app/components/redesign/components/revshare/ShareInput.tsx

```diff
@@ -125,15 +238,24 @@ export const ShareInput = React.memo(
               onChangePercent(Number(e.target.value) / 100)
             }
             disabled={percentDisabled}
+            aria-disabled={percentDisabled}
+            aria-describedby={`percent-description-${index}`}
           />
+          <div id={`percent-description-${index}`} className="sr-only">
+            Calculated from weight, or can be edited directly.
+          </div>
         </div>
-        <div className="hidden md:block">
+        <div
+          role="cell"
+          className="hidden md:block"
+          aria-labelledby="col-action"
+        >
           <ToolsSecondaryButton
             onClick={onRemove}
             className="border-none py-sm px-xs shrink-0"
-            aria-label={`Remove recipient ${index + 1}`}
+            aria-label={`Remove recipient ${index + 1} from revenue sharing table`}
```

Comment:
Instead of referencing by index, we should be able to reference the wallet address input. So, the label for button can be "Remove recipient" and we can `aria-describedby` to the input for wallet address.